apps/io-wallet-support-func/openapi.yaml

```diff
@@ -0,0 +1,220 @@
+openapi: 3.0.3
+info:
+  title: IO Wallet - Support API
+  version: 1.0.0
+servers:
+  - url: https://io-p-itn-wallet-support-func-01.azurewebsites.net/api/v1/wallet
+paths:
+  /wallet-instances/current/status:
+    post:
+      summary: Retrieve the current Wallet Instance status given a fiscal code
+      operationId: getCurrentWalletInstanceStatusByFiscalCode
+      security:
+        - FunctionsKey: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/GetCurrentWalletInstanceStatusByFiscalCodeBody"
+      responses:
+        "200":
+          description: Wallet Instance status successfully retrieved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/WalletData"
```

Comment:
```suggestion
                $ref: "#/components/schemas/WalletInstanceStatus"
```

Reply:
we'll leave `WalletData` as said in the comment below